src/riakc_ts_query_operator.erl

```diff
@@ -47,17 +47,26 @@ serialize_interpolations([{Key, Value} | RemainingInterps],
     UpdatedInterps = [#rpbpair{key=Key, value=Value} | SerializedInterps],
     serialize_interpolations(RemainingInterps, UpdatedInterps).
 
-deserialize({error, {Code, Message}}) when is_integer(Code), is_list(Message) ->
+deserialize(Response) ->
+    deserialize(Response, false).
+
+deserialize({error, {Code, Message}}, _Types) when is_integer(Code), is_list(Message) ->
```

Comment:
Or even _ColumnTypes?

On Mon, Jul 25, 2016 at 12:17 PM, Luke Bakken <notifications@github.com>
wrote:

> In src/riakc_ts_query_operator.erl
> <https://github.com/basho/riak-erlang-client/pull/303#discussion_r72094980>
> :
>
> > @@ -47,17 +47,26 @@ serialize_interpolations([{Key, Value} | RemainingInterps],
> >      UpdatedInterps = [#rpbpair{key=Key, value=Value} | SerializedInterps],
> >      serialize_interpolations(RemainingInterps, UpdatedInterps).
> >
> > -deserialize({error, {Code, Message}}) when is_integer(Code), is_list(Message) ->
> > +deserialize(Response) ->
> > +    deserialize(Response, false).
> > +
> > +deserialize({error, {Code, Message}}, _Types) when is_integer(Code), is_list(Message) ->
>
> Can _Types be renamed to _IncludeColumnData to make the intent clearer?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/basho/riak-erlang-client/pull/303/files/c1ac1b50288bba338e6eb276455d15ab186ef224#r72094980>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAFnqOtKvOkRCGywr4WNoKlUmCkjX2rks5qZOGjgaJpZM4JSZA2>
> .
>